build.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
+set -e
 
 if hash powershell 2>/dev/null; then
     echo 'Continuing with `powershell -noprofile -c Start-PSBuild`'
     powershell -noprofile -c "Import-Module ./build.psm1; Start-PSBuild"
 else
-    echo 'No `powershell`, see docs/building/linux.md or macos.md to build PowerShell!'
+   echo 'Continuing with full manual build'
+   ./build-from-scratch.sh
```

Comment:
Again, I don't think it should be run automatically.
The error message should suggest to use this script, but not run it.